atom/app/manifests.cc

```diff
@@ -25,6 +28,25 @@ const service_manager::Manifest& GetElectronContentBrowserOverlayManifest() {
           .RequireCapability("proxy_resolver", "factory")
           .RequireCapability("chrome_printing", "converter")
           .RequireCapability("pdf_compositor", "compositor")
+          .RequireCapability(content::mojom::kRendererServiceName,
+                             "electron_api")
+          .Build()};
+  return *manifest;
+}
+
+const service_manager::Manifest& GetElectronContentRendererOverlayManifest() {
+  static base::NoDestructor<service_manager::Manifest> manifest{
+      service_manager::ManifestBuilder()
+          .WithDisplayName("Electron (renderer process)")
+          .ExposeCapability("electron_api",
+                            std::set<const char*>{
+                                "electron_api.mojom.Electron",
+                            })
+          .ExposeInterfaceFilterCapability_Deprecated(
+              "navigation:frame", "browser",
+              std::set<const char*>{

```

Comment:
I think its preferred to use `service_manager::Manifest::InterfaceList<T>()` variation for this ?

Reply:
Huh, TIL. Interestingly, this code seems to work fine now when I remove this stuff from the manifest. Not sure what changed.

Reply:
Restored part of this due to changing how I'm binding the `ElectronBrowser` service and used `Manifest::InterfaceList`. Thanks for the pointer.